graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/buffers/MessageEvent.java

```diff
@@ -45,6 +46,7 @@ public MessageEvent newInstance()
 
     private RawMessage raw;
     private Message msg;
```

Comment:
Maybe we should just have two different MessageEvent classes. One for the ProcessBuffer with a collection of messages and one for the OutputBuffer with a single message.